bob/src/hw_metrics_collector.rs

```diff
@@ -179,53 +184,100 @@ enum CommandError {
     Primary(String),
 }
 struct CPUStatCollector {
-    iostat_avl: bool,
+    procfs_avl: bool,
 }
 
+const CPU_IOWAIT_COLUMN: usize = 5;
```

Comment:
This const is only used in one function, so I suggest declaring it directly inside the function body.